docs/semgrep-app/findings.md

```diff
@@ -124,9 +122,10 @@ To **open findings**:
 4. Click **Open**.
 5. Optional: Include a **comment** explaining the action.
 
-To **close findings**, do one of the two options:
+To **fix a finding**, update, or refactor the code such that the Semgrep rule pattern no longer matches the code.
```

Comment:
This is great! Can we also add something like:
```suggestion
To **fix a finding**, update, or refactor the code such that the Semgrep rule pattern no longer matches the code.

There are two ways to **remove a finding** altogether: either remove the rule that found it or remove the file it is in from the list of files scanned by semgrep. Then, re-scan the branch or pull request containing the issue. The issue should then be removed.
```

Reply:
(Have added this in!)